src/ui/post_card/small_post_card/small_post_card.tsx

```diff
@@ -0,0 +1,150 @@
+    setIsLiked(!isLiked);
+    setIsDisliked(false);
+    setAmountOfLikes(isLiked ? amountOfLikes - 1 : amountOfLikes + 1);
+  }
+  return (
+    <SmallPostCardWrapper>
+      <SmallMainWrapper>
+        <SmallCardtextWrapper>
+          <SmallPostCardDate>{props.postCard.date}</SmallPostCardDate>
+          <SmallPostCardTitle>{props.postCard.title}</SmallPostCardTitle>
+        </SmallCardtextWrapper>
+        <SmallCardImageWrapper>
+          <img src={props.postCard.image} alt="#"></img>
+        </SmallCardImageWrapper>
+      </SmallMainWrapper>
+      <SmallIconWrapper>
+        <SmallLikeWrapper>
+          <LikeIcon onClick={() => like()}>
+            {isLiked ? (
+              <i className="fa-solid fa-thumbs-up"></i>
+            ) : (
+              <i className="fa-regular fa-thumbs-up"></i>
+            )}
+          </LikeIcon>
+          <AmountOfLikes>{amountOfLikes}</AmountOfLikes>
+          <DislikeIcon onClick={() => dislike()}>
+            {isDisliked ? (
+              <i className="fa-solid fa-thumbs-down"></i>
+            ) : (
+              <i className="fa-regular fa-thumbs-down"></i>
+            )}
+          </DislikeIcon>
+        </SmallLikeWrapper>
+        <SmallSaveIcoonWrapper>
+          <BookmarkIcon onClick={() => setIsSaved(!isSaved)}>
+            {isSaved ? (
+              <i className="fa-solid fa-bookmark"></i>
+            ) : (
+              <i className="fa-regular fa-bookmark"></i>
+            )}
+          </BookmarkIcon>
+          <i className="fa-solid fa-ellipsis"></i>
+        </SmallSaveIcoonWrapper>
+      </SmallIconWrapper>
+    </SmallPostCardWrapper>
+  );
+};
+
+const SmallPostCardWrapper = styled.div`
+  width: 350px;
+  background-color: #f3f3f3;
+  padding: 10px 20px;
+  border-bottom: 1px solid #cfd0d1;
+  height: 157px;
+`;
+const SmallMainWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 15px;
+`;
+const SmallCardtextWrapper = styled.div`
+  width: 75%;
+`;
+const SmallPostCardDate = styled.div`
+  color: #8b8a90;
+  font-size: 16px;
+  height: 18px;
+`;
+
+const SmallPostCardTitle = styled.h2`
+  color: #403f44;
+  line-height: 26px;
+  font-size: 22px;
+  margin: 10px 0;
+`;
+
+const SmallCardImageWrapper = styled.div`
+  width: 80px;
+  height: 80px;
+  & img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    object-position: center;
+  }
+`;
+const SmallIconWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 26px;
+`;
+const SmallLikeWrapper = styled.div`
+  width: 20%;
+  display: flex;
+  justify-content: space-between;
+  font-size: 20px;
+  align-items: center;
+`;
+const SmallSaveIcoonWrapper = styled.div`
+  width: 20%;
+  display: flex;
+  justify-content: space-between;
+  font-size: 20px;
+`;
+
+const BookmarkIcon = styled.div`
+  display: flex;
+  justify-content: space-between;
+  font-size: 20px;
+`;
+
+const LikeIcon = styled.div`

```

Comment:
common styled components can be moved to separated files - this way you can reuse them without copy-pasting